README.md

```diff
@@ -8,10 +8,20 @@ To set up a dev instance of StreamWebs, copy ``Dockerfile.env.dist`` to
 ``Dockerfile.env``. Then, run the following:
 ```
 $ sudo docker-compose build
-$ sudo docker-compose run web
+$ sudo docker-compose up
```

Comment:
I would replace all instances of `sudo docker-compose` with just `docker-compose`. People should probably know if they need to run docker commands as root or not.